src/org/usfirst/frc/team467/robot/Autonomous/ActionGroup.java

```diff
@@ -1,5 +1,7 @@
 package org.usfirst.frc.team467.robot.Autonomous;
 
+import java.util.ArrayList;
```

Comment:
I'm not sure, but I don't think you need ArrayList anymore. At least, not once you fully convert to varargs form.